packages/shared/src/components/marketingCta/MarketingCtaPopoverSmall.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { ReactElement, useCallback, useEffect, useRef } from 'react';
+import classNames from 'classnames';
+import { Card } from '../cards/Card';
+import { CardCover } from '../cards/common/CardCover';
+import { CTAButton, Description, MarketingCta, Title } from './common';
+import { ViewSize, useBoot, useViewSize } from '../../hooks';
+import { useLogContext } from '../../contexts/LogContext';
+import { LogEvent, TargetType } from '../../lib/log';
+import { ButtonSize, ButtonVariant } from '../buttons/common';
+import { MiniCloseIcon } from '../icons';
+import { Button } from '../buttons/Button';
+
+export function MarketingCtaPopoverSmall({
+  marketingCta,
+}: {
+  marketingCta: MarketingCta;
+}): ReactElement {
+  const { title, description, image, ctaUrl, ctaText } = marketingCta.flags;
+  const { clearMarketingCta } = useBoot();
+  const { logEvent } = useLogContext();
+  const isImpressionTracked = useRef(false);
+  const isMobile = useViewSize(ViewSize.MobileL);
+
+  useEffect(() => {
+    if (isImpressionTracked.current) {
+      return;
+    }
+
+    logEvent({
+      event_name: LogEvent.Impression,
+      target_type: TargetType.MarketingCtaPopoverSmall,
+      target_id: marketingCta.campaignId,
+    });
+    isImpressionTracked.current = true;
+  }, [marketingCta.campaignId, logEvent]);
+
+  const onCtaClick = useCallback(() => {
+    logEvent({
+      event_name: LogEvent.Click,
+      target_type: TargetType.MarketingCtaPopoverSmall,
+      target_id: marketingCta.campaignId,
+    });
+    clearMarketingCta(marketingCta.campaignId);
+  }, [clearMarketingCta, marketingCta.campaignId, logEvent]);
+
+  const onCtaDismiss = useCallback(() => {
+    logEvent({
+      event_name: LogEvent.MarketingCtaDismiss,
+      target_type: TargetType.MarketingCtaPopoverSmall,
+      target_id: marketingCta.campaignId,
+    });
+    clearMarketingCta(marketingCta.campaignId);
+  }, [clearMarketingCta, marketingCta.campaignId, logEvent]);
+
+  return (
+    <Card
+      className={classNames(
+        '!p-0',
+        isMobile
+          ? '!max-h-none !border-none !bg-transparent !shadow-none'
```

Comment:
I re-used the `Card` component here since it was pretty much close to what the design was.

With this one, some of the overrides on mobile is because it's a drawer, and not a card. Though for the other, we could maybe add a new prop to `Card` that lets us select the border colour

Reply:
Yeah that could be one, but as we see the overrides here are not just the border.

Reply:
I could extract the classes I need from `Card` component, and not re-use it? Since it's not actually a card?

Reply:
Yeah I think could be better, it's a totally different component

Reply:
Will do that then 🫡

Reply:
Yeah maybe conditional render the Card (or even some other name?) or Drawer?
(Maybe having `PopupCard` would work?